### PR TITLE
40 footer component (layout w/o wave part)

### PR DESCRIPTION
**Pull request by natapokie**
footer looks like this on desktop
image

on mobile, 
image

site map links should take you to the corresponding pages and social media icons should take you to the corresponding social links

### CODE REVIEW

File: client/src/components/footer/FooterStorybook.jsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Footer.scss';
+
+import FacebookIcon from '../../assets/social/facebook-square-brands.svg';
+import InstagramIcon from '../../assets/social/instagram-brands.svg';
+import TwitterIcon from '../../assets/social/twitter-brands.svg';
+import SnapchatIcon from '../../assets/social/snapchat-brands.svg';
```
Comment (jameskokoska) [inline client/src/components/footer/FooterStorybook.jsx:8]: If possible use a snapchat filled icon? might look a bit more consistent

File: client/src/components/footer/FooterStorybook.jsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Footer.scss';
+
+import FacebookIcon from '../../assets/social/facebook-square-brands.svg';
+import InstagramIcon from '../../assets/social/instagram-brands.svg';
```
Comment (jameskokoska) [inline client/src/components/footer/FooterStorybook.jsx:6]: Same here, if you can find a more filled icon

File: client/src/components/footer/Footer.scss [hunk 3]
```diff
@@ -0,0 +1,127 @@
+@import '../../scssStyles/_mixins.scss';
+
+.footer-container {
```
Comment (jameskokoska) [inline client/src/components/footer/Footer.scss:3]: Can you add some horizontal padding? Maybe just for desktop
So stuff is pushed away from the side

**Review by jameskokoska — CHANGES_REQUESTED** [review]
Looks really nice! I like the mobile view too

**Review by jameskokoska — APPROVED** [review]

**Comment by Cali0707** [comment]
@natapokie can you resolve the merge conflicts here?

**Comment by natapokie** [comment]
> @natapokie can you resolve the merge conflicts here?

@Cali0707 resolved!

File: client/src/util/socials.jsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import FacebookIcon from '../assets/social/facebook-square-brands.svg';
+import InstagramIcon from '../assets/social/instagram-square-brands.svg';
+import TwitterIcon from '../assets/social/twitter-square-brands.svg';
+import SnapchatIcon from '../assets/social/snapchat-square-brands.svg';
+import YoutubeIcon from '../assets/social/youtube-square-brands.svg';
```
Comment (Cali0707) [inline client/src/util/socials.jsx:5]: @natapokie it looks like this PR is adding 9 SVGs but only importing 5 - can you remove any unused ones?

**Review by Cali0707 — CHANGES_REQUESTED** [review]
Looks great- one small fix and then this should be ready to merge!

File: client/src/util/socials.jsx (hunk 4 above)
Comment (natapokie) [reply client/src/util/socials.jsx:5]: Yep! I imported the svg files again in the storyboard version of the footer, i just removed them so it should be good now!

**Review by Cali0707 — APPROVED** [review]

**Pull request merged**